doc/api/assert.markdown

```diff
@@ -2,10 +2,10 @@
 
     Stability: 3 - Locked
 
-The `assert` module provides a simple set of assertion tests that can be used
-to test invariants and implement unit tests. While the `assert` module is
-generally intended for internal use by Node.js itself, it can be used by user
-code calling `require('assert')`.
+The `assert` module provides a simple set of assertion tests that can be used to 
+test invariants. `assert` is intended for internal use by Node.js, but can be 
+used in application code via `require('assert')`. `assert` is not a testing 
+framework, and is not intended to be used as a general purpose assertion library.
 
```

Comment:
Minor nit... I'd like to avoid starting sentences with a lower-cased module name, especially two sentences in a row... Perhaps reword the las two sentences to:

```
The module is intended for internal use by Node.js, but can be used in application 
code via `require('assert')`. However, `assert` is not a testing framework, and is not 
intended to be used as a general purpose assertion library.
```